src/PhpPact/Consumer/Registry/Interaction/Body/AbstractBodyRegistry.php

```diff
@@ -38,7 +38,6 @@ function (bool $success, Part $part) use ($body) {
                 },
                 true
             ),
-            default => false,
```

Comment:
With the removal of a default, what happens if `$body` is _not_ an instance of either of the classes being tested?

Reply:
It can't be. The type is defined for `$body` already make sure that it must be instance of those classes:

```
public function withBody(Text|Binary|Multipart $body): void
```

This removal is a suggestion by the `PHPStan` tool. If I don't remove it, this PR will failed.